templates/haproxy.cfg.j2

```diff
@@ -37,7 +37,9 @@ defaults
 
 {% for front in haproxy_frontends %}
 frontend {{ front.name }}
-    bind {{ front.address }}
+{%   for adr in front.address %}
```

Comment:
Could you rename this var `addr`?

Reply:
addr: sure/done :) 

semantic versioning: did that already, seems to me, that the role is now in a decent shape. the only thing I might be working on at a later time would be adding support for remote logging (should be a non-breaking change). I might also work on a wrapper role, that would setup Heartbeat+HAproxy+DNS server to get a high-availability setup. No changes for this role should be required to achieve that as well.